ddtrace/llmobs/_llmobs.py

```
@@ -553,7 +556,7 @@ def _patch_integrations() -> None:
         integrations_to_patch.update(
             {k: asbool(v) for k, v in dd_patch_modules_to_str.items() if k in SUPPORTED_LLMOBS_INTEGRATIONS.values()}
         )
-        patch(**integrations_to_patch)
+        patch(raise_errors=True, **integrations_to_patch)
```

what's the context for adding this? is there a case where our use of `patch` could lead to patching errors? 

totally fine w this, but wondering if maybe we should also make a note of it in the rel note since if there was a case where this was happening for users before, it'll now throw instead of log (which tbf is probably what we should've done from the get-go anyways), which could be noteworthy - wdyt?

---

>def patch(raise_errors=True, **patch_modules):
The old default for the kwarg should be `True` so i wouldnt expect any behavior changes! i can't think of cases where this would be breaking since this is code inside enable() that we control

I added this since we are now passing in `integrations_to_patch` with both `true/false` argument values **and** a list of modules for botocore, resulting mypy errors since it expects `raise_errors` to always be a bool 

this allows us to be more explicit in setting the `raise_errors` to `True`

---

Let's move this to a separate PR, since it's a separate change with different implications.

---

discussed briefly offline but since this change is only to fix a type check and doesn't result in any behavior changes, going to close this thread - will raise an issue separately on if we should be raising errors or not on `LLMObs.enable()`